src/writer.rs

```diff
@@ -127,6 +131,12 @@ impl<'a> WriterOptions<'a> {
         self
     }
 
+    /// Sets an extension sequence for the writer
+    pub fn extension_sequence(mut self, extension_sequence: Option<ExtensionSequence>) -> Self {
```

Comment:
Normally, this would be something like `with_extensions` to follow the builder pattern.

Reply:
Done